src/main/antlr4/imports/COMMAND_REPLACE_MODE.g4

```diff
@@ -76,7 +76,7 @@ fragment COMMAND_REPLACE_MODE_CHAR
          // |'\u0029' // )
          |'\u002A' // *
          |'\u002B' // +
-         |'\u002C' // ,
+         // |'\u002C' // ,
```

Comment:
I am not familiar with the project but why commented out instead of removed?

Reply:
It's essentially a list of all characters usually used in Strings in the grammar, so I'd say it's faster to see which characters are not allowed in this specific command by looking at what's commented out.